Core/Application.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Core.Configuration;
+using DocoptNet;
+using JsonNet.ContractResolvers;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Schema.Generation;
+
+namespace Core
+{
+    internal static class Application
+    {
+        private const string Usage = @"
+Usage:
+  core run [--config=<file>]
+  core testconfig [--config=<file>]
+  core gendefaultconfig [--outfile=<file>]
+  core regenjsonschema
+  core -h | --help
+
+Options:
+  -h --help         Show this screen.
+  --config=<file>   Specifies the config file to use. [default: config.json]
+  --outfile=<file>  Specifies the file to output to. Default is printing to stdout.
+";
+
+        private static void Main(string[] args)
+        {
+            using var loggerFactory = LoggerFactory.Create(builder => builder
+                .AddFile("logs/tpp-{Date}.log")
```

Comment:
Logger file is not configurable? Maybe we could have it log only to console until it gets a log destination in the config